.gitlab-ci.yml

```diff
@@ -32,13 +32,13 @@ stages:
         --create-namespace
         --namespace $KUBE_NAMESPACE
         --set image.backend.repository="${BACKEND_CONTAINER_REPO}"
-        --set image.backend.tag="${CI_COMMIT_SHORT_SHA}-beta"
+        --set image.backend.tag="${CI_COMMIT_SHORT_SHA}-beta-leaktests"
         --set image.frontend.repository="${FRONTEND_CONTAINER_REPO}"
-        --set image.frontend.tag="${CI_COMMIT_SHORT_SHA}-beta"
+        --set image.frontend.tag="${CI_COMMIT_SHORT_SHA}-beta-leaktests"
```

Comment:
these should also be reverted

Reply:
I didn't want images being built on this branch to conflict/overwrite those that are built and deployed from master, I guess. I don't mind either way though, so happy to revert :)

Reply:
actually, all the images would be unique anyways as their tag contains ${CI_COMMIT_SHORT_SHA} which is supposed to be unique. So the pipeline wouldn't overwrite the existing images.